### PR TITLE
Updating documentation

### PR DESCRIPTION
**Pull request by Tocseoj**
- changed incorrect parenthetical
- added Tera type to packed format description

### CODE REVIEW

File: sim/TEAMS.md [hunk 1]
```diff
@@ -210,7 +210,7 @@ NICKNAME|SPECIES|ITEM|ABILITY|MOVES|NATURE|EVS|GENDER|IVS|SHINY|LEVEL|HAPPINESS,
 
 - `DYNAMAXLEVEL` is left blank for 10.
 
-- If `POKEBALL`, `HIDDENPOWERTYPE`, `GIGANTAMAX` and `DYNAMAXLEVEL`
+- If `POKEBALL`, `HIDDENPOWERTYPE`, `GIGANTAMAX`, `DYNAMAXLEVEL` and `TERATYPE`
```
Comment (MathyFurret) [inline sim/TEAMS.md:213]: It should be explained that leaving TERATYPE blank defaults it to the Pokemon's first type

Comment (Tocseoj) [reply sim/TEAMS.md:215]: @MathyFurret Thanks for the review. I've made a related note and this PR should be good to be merged.

File: sim/TEAMS.md [hunk 2]
```diff
@@ -210,7 +210,9 @@ NICKNAME|SPECIES|ITEM|ABILITY|MOVES|NATURE|EVS|GENDER|IVS|SHINY|LEVEL|HAPPINESS,
 
 - `DYNAMAXLEVEL` is left blank for 10.
 
-- If `POKEBALL`, `HIDDENPOWERTYPE`, `GIGANTAMAX` and `DYNAMAXLEVEL`
+- `TERATYPE` if left blank, it will default to the pokemon's first type.
```
Comment (MathyFurret) [inline sim/TEAMS.md:213]:
```suggestion
- `TERATYPE` if left blank will default to the Pokémon's first type.
```

**Review by MathyFurret — APPROVED** [review]

**Comment by DaWoblefet** [comment]
Thanks Tocseoj!

**Pull request merged**